data/856/327/11/85632711.geojson

```diff
@@ -228,8 +249,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1587428798,
-    "wof:name":"United Nations Disengagement Observer Force Zone",
+    "wof:lastmodified":1616015335,
+    "wof:name":"United Nations Disengagement Observer Force",
```

Comment:
This geometry could use some love.

Reply:
This record was last updated when Israel and Palestine area geometries were updated, so this geometry is aligned with country/region/county/other disputed geometries already.

If it works for you, I'll likely keep this geometry as-is in this PR, then take another pass at this entire area in a follow-up PR that adjusts countries/point-of-view geometries.

Reply:
On second though, now that I'm working through country geometry updates, I think it makes more sense to keep this PR open and include a change to this record's geometry. I'll add a commit to update this record's geometry.

Reply:
So I've gone through all disputed and country records and aligned geometries across placetypes. This record actually aligns well and looks correct - maybe we can chat about this specific record together?